src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsInterstitialObject.java

```diff
@@ -30,17 +33,30 @@
   @FindBy(css = "iframe.wikia-ad-iframe")
   private WebElement interstitialAdIframe;
 
-  @FindBy(css = "div#ext-wikia-adEngine-template-modal, div.lightbox-content-inner > div")
+  static final String interstitialAdWrapperSelector = "div#ext-wikia-adEngine-template-modal, div.lightbox-content-inner > div";
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS00115)